pkg/storage/scan.go

```diff
@@ -190,13 +196,13 @@ func (s *PostgreSQLStore) GetScans(tenantID string) ([]*domain.ScanSummary, erro
       COALESCE(A.medium, 0) AS medium,
       COALESCE(A.high, 0) AS high,
       COALESCE(A.critical, 0) AS critical
-   FROM  scans S
+   FROM  (SELECT * FROM scans WHERE status!=$2) S
    INNER JOIN hosts H ON S.host_id = H.id
    LEFT JOIN aggregated_vulnerabilities A ON S.id = A.scan_id
    WHERE S.tenant_id = $1
    ORDER BY S.started_at DESC`
```

Comment:
Performance wise, I think it might be better to leave this conditional in the WHERE clause:

 ```suggestion
   FROM scans  S
   INNER JOIN hosts H ON S.host_id = H.id
   LEFT JOIN aggregated_vulnerabilities A ON S.id = A.scan_id
   WHERE S.tenant_id = $1 AND S.status != $2
   ORDER BY S.started_at DESC`
```